### ISSUE TITLE
Palettes expanded/collapsed state not persisted

### ISSUE THREAD

**Issue by j-serda** [musescore/MuseScore#17064]
### Issue type

UI bug

### Bug description

The state of the palettes (whether they are expanded or collapsed) in the UI does not persist when Musescore is closed, meaning that every time the application is launched, each palette needed to work (or all of them if using the _Expand all Palettes_ option of the _Palettes_ pane) needs to be manually expanded, which is time consuming and frustrating.

Back in Musescore 3.X, it used to be that the state of the palettes would persist between sessions, meaning that the program would reopen with the palettes in the state in which they were left when closed. Either this is a bug in Musescore 4.X, which should be corrected, or this is a wanted behavior, in which case a good middle ground could be to implement a "_Palette persistance_" option in the parameters to let users choose wether or not they want their palettes state to persist between sessions.

### Steps to reproduce

1. Open Musescore and create a new score (or open an existing one)
2. In the _Palettes_ pane options, click on _Expand all Palettes_
3. Close Musescore
4. Reopen Musescore and reopen the previous score
5. All the palettes are once again collapsed

### MuseScore Version

4.0.2.230651546

### Regression

Yes, this used to work in Musescore 3.x and now is broken

### Operating system

macOS 10.15.7

### Additional context

OS: macOS 10.15
Arch.: x86_64
MuseScore version (64-bit): 4.0.2-230651546
revision: github-musescore-musescore-dbe7c6d

**Comment by bkunda** [comment]
Are we talking here about whether one or more palettes _remain_ _open_ or not between sessions? 
If this is the case, then this information should be stored to the file, because it wouldn’t always make sense for the same palettes to remain open between different projects.

**Comment by j-raybaudserda** [comment]
Precisely, it used to be that the palettes that were opened during a session remained open between the sessions if not closed manually (and the closed ones stayed closed as well).
Given the fact that the list of palettes does not change between sessions (except if modified manually, which makes it non project dependant), it seemed fitting to me to have a global option to control this behaviour.

**Comment by FranckDhersin** [comment]
Hello, 
I have reported the bug #19450.
I agree with you @cbjeukendrup but this information (the Palettes Display Status) should be stored somewhere in the personal computer (maybe in a file visited whenever an instance of MS is opened or closed).

I work on a lot of scores and each time I open Musescore, it is convenient to always have the same Palettes-Submenus opened.
So that I don't have to open it each time I want a tempo or an articulation or whatelse.

**Comment by lgvr123** [comment]
Why not having an option to save the current scheme of opened/closed palettes as the default scheme when MU is opened ?

**Comment by fernandomartin777** [comment]
I confirm that it worked in MU 3.x and doesn't work in MU 4.x, being a regression. Please, bring this feature back.
